packages/pangea-sdk/PangeaCyber.Net/Audit/Requests/ResultsRequest.cs

```diff
@@ -54,6 +61,13 @@ public Builder WithOffset(int? offset)
                 return this;
             }
 
+            ///
+            public Builder WithReturnContext(bool? rc)
```

Comment:
I know the other methods use nullable params but since this is a new one, does it even make sense to accept null? 
```suggestion
            public Builder WithReturnContext(bool rc)
```